etcd/restore-etcd-cluster-on-coreos.md

```diff
@@ -0,0 +1,110 @@
+# Restore etcd Cluster on CoreOS
+
+Here you will find step-by-step instructions on how to recover your etcd cluster. It is important to know that you can not restore your etcd cluster using only discovery URL. Discovery URL is used only once at bootstrap.
+
+In this example we will review 3-members etcd cluster with one failed node. It is important to take into consideration that this example supposes you used [Cloud-Config][cloud-config] with [discovery URL][etcd-discovery] to bootstrap your cluster with the following default options:
+
+```yaml
+  etcd2:
+    advertise-client-urls: http://%PEER_IP_ADDRESS%:2379
+    initial-advertise-peer-urls: http://%PEER_IP_ADDRESS%:2380
+    listen-client-urls: http://0.0.0.0:2379,http://0.0.0.0:4001
+    listen-peer-urls: http://0.0.0.0:2380
+    discovery: %DISCOVERY_URL%
+```
+
+If you have etcd cluster with TLS, just use `https://` instead of `http://` in commands' examples below. Make sure you didn't set custom etcd [member name][etcd-member-name] (by default CoreOS uses [machine-id] for etcd member name). Let's assume that your etcd cluster has faulty `172.16.0.102` member:
+
+```sh
+$ etcdctl cluster-health
+member fe2f75dd51fa5ff is healthy: got healthy result from http://172.16.0.101:2379
+failed to check the health of member 1609b5a3a078c227 on http://172.16.0.102:2379: Get http://172.16.0.102:2379/health: dial tcp 192.168.122.69:2379: connection refused
+member 1609b5a3a078c227 is unreachable: [http://172.16.0.102:2379] are all unreachable
+member 60e8a32b09dc91f1 is healthy: got healthy result from http://172.16.0.103:2379
+cluster is healthy
+```
+
+First of all `etcdctl` on failed node should be configured to use healthy endpoint, so we have to configure environment variable and you will be able to run `etcdctl` command on your failed node:
```

Comment:
Can we sidestep 27-31 if we just say, "run etcdctl from a working node"?

This also brings up a bigger question, what type of failure are we trying to recover from here? If you can get on the node, I would imagine that is a disk failure or similar. We should probably state this at the top.